bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+
+
+### Effects on Borrowers of a GS-Protected Asset
+
+There are multiple effects on _borrowers of a GS-protected_ asset versus a globally settled asset.
+
+#### Old Debt Positions
+
+GSP ensures that the blockchain keeps old debt positions open and ensures that:
+
+- the borrower still owes the blockchain the original debt/smart asset, and
+- the borrower still owns the collateral behind each debt position that is held in escrow by the blockchain.
+
+This is both beneficial and detrimental to old borrowers and depends on what happens while GSP is activated.
+
+#### <div id="risk-to-old-borrowers" /> During GSP
+
+GSP activation provides time to old borrowers whose debt positions were openly under margin call threat to supply more collateral and increase their collateral ratio above the minimum collateral ratio.  _This is a benefit to the borrower **if the borrower can provide more collateral** before the margin call is completed.  This is an opportunity to secure their prior collateral by adding new collateral._
+
+|Comparison to Globally Settled Asset|
+|-|
+|Debt owed to the blockchain are closed by the blockchain confiscating enough collateral to equal the debt in a position at the global swan price.  This confiscated collateral is moved to the settlement fund.  The excess collateral is returned to the borrower.<p><p>_Under GSP, **if the borrower cannot provide enough collateral** before the margin call is completed, then the borrower will lose more collateral than under global settlement._|
+
+#### <div id="risk-to-old-borrowers-after-gsp" /> After GSP
+
+_If_ the borrower can avoid having his or her debt position margin called during the GSP period, then the borrower's previous collateral is secured.
+
+|Comparison to Globally Settled Asset|
+|-|
+|There are no old borrowers after a [globally settled settle is revived per BSIP-18](https://github.com/bitshares/bsips/blob/master/bsip-0018.md) because old debt positions are closed when global settlement is triggered.|
+
+#### <div id="gsp-activated-new-debt" />New Debt Positions
+
+Creating/borrowing new debt positions are still possible under GSP.  Borrowing new debt positions still requires a multiple of the backing asset based on the MCR multiplier.  The debt-denominated price of the total collateral behind the debt position equals
+
+K<sup>D</sup> =  K &times; FP<sub>P</sub><sup>K/D</sup>
+
+The maximum debt that can be obtained for that collateral is
+
+D &le; D<sub>max</sub> = K<sup>D</sup> &div; MCR
+
+or
+
+D &le; D<sub>max</sub> = K &times; FP<sub>P</sub><sup>K/D</sup> &div; MCR
+
+But the valuation of that collateral is based on the protected feed price FP<sub>P</sub> which is by definition higher than the natural price of the backing asset on external market.
+
+FP<sub>P</sub> = max(FP<sub>N</sub>, P<sub>GSP</sub>)
+
+This has the following implications:
+
+- _The true price of the collateral backing a new debt position is worth less than if that natural price were used._
```

Comment:
Hm, good point. Perhaps the BSIP should specify that the CR must be increased during GSP so that the true collateral value is backing the debt at the intended CR (e. g. 175%). (This rule can take effect only after https://github.com/bitshares/bitshares-core/issues/1270 has been resolved.)